src/groups/mqb/mqba/mqba_domainmanager.cpp

```diff
@@ -686,19 +686,25 @@ int DomainManager::processCommand(mqbcmd::DomainsResult*        result,
             }
         }
 
-        DecodeAndUpsertValue unused;
+        DecodeAndUpsertValue configureResult;
         d_configProvider_p->clearCache(domainSp->name());
         d_configProvider_p->getDomainConfig(
```

Comment:
No need to do anything, just my observation here

Note that docs for `getDomainConfig` are incorrect. They state that the method is asynchronous:
```
    /// Asynchronously get the configuration for the specified `domainName`
    /// and invoke the specified `callback` with the result.
```

In fact, this method is synchronous and thread safe, protected by mutes.
It means that you are safe to access `configureResult` just after the `getDomainConfig` call because you are guaranteed that the method was already executed.

I will update the docs later.